cypress/integration/Content.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Content component
+  I want to change Content component properties
+
+  Background: Open Content component page
+    Given I open "Content" component page
+
+  @positive
+  Scenario Outline: Change data in Content dialog
```

Comment:
those steps should be separeted to several scenarios to make them independent tests

Reply:
fixed